mist-api/src/lib/util.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+// Vendor
+const _ = require('lodash')
+
+// Local
+const util = require('core-lib/util')
+
+exports.splitIntoQueryTerms = (value) => {
+    return util.splitIntoTerms(value).map((term) => `%${term}%`)
+}
+
+exports.checkQueryAndTarget = (queryValue, target) => {
+    queryValue = queryValue && queryValue.trim()
+    if (!queryValue)
+        return
+    if (!target)
+        throw new Error('Must have a defined target')
+    return queryValue;
+}
+
+exports.assignExactMatchCriteria = (queryValue, target, fields) => {
+    queryValue = exports.checkQueryAndTarget(queryValue, target)
+    const terms = exports.splitIntoQueryTerms(queryValue)
+    if (terms.length > 0) {
+        if (fields) {
+            fields
+            .forEach((fieldName) => {
+                _.set(target, `criteria.where.$or.${fieldName}`, queryValue)
```

Comment:
`queryValue` should be `terms`.

Also for improved security, please replace all string operators (these are now deprecated) with their sequelize aliases. See: http://docs.sequelizejs.com/manual/tutorial/querying.html#operators-aliases

Reply:
I will call here `util.splitIntoTerms` instead of exports.splitIntoQueryTerms (which adds % to terms for iLike operator) to produce terms since this is exact match search.